### PR TITLE
Add Dockerfile to run test_parser

### PR DESCRIPTION
**Pull request by steren**
This will be helpful to develop new parser or run parser tests in an environment that might not have all dependencies (see #4326 as an example)

### CODE REVIEW

File: parsers.dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Container image that executes the parser tests 
+# Build with "$ docker build -f parsers.dockerfile -t parsers . "
+# Run with  "$ docker run parsers US-CAL-CISO production"
+# If needed, pass any token env var using "docker run -e TOKEN=<token>"
+
+FROM python:3.10
```
Comment (VIKTORVAV99) [inline parsers.dockerfile:6]:
```suggestion
FROM python:3.8
```
This should be python 3.8

**Review by VIKTORVAV99 — COMMENTED** [review]
I won't take a stance on if this should be merged or not, that is up to the team but I think the plan is to move away from docker, at least for the frontend.

But in either case it should use python 3.8 or resolve the `.python-version` file.

**Comment by Kongkille** [comment]
Thanks, yeah we're intending to move away from docker on the frontend, but it might still be useful for the parsers.

Adding @FelixDQ

**Review by FelixDQ — COMMENTED** [review]
The backend for the parsers still uses docker (In Cloud Run), and we don't intend to move away from that. The environment can be quite annoying to set up, so a docker file for contributors seems like a great idea! We just need to remember to update the documentation in the wiki as well.

@steren can you add these to the apt-get install as well: `apt-get install -y tesseract-ocr tesseract-ocr-jpn libgl1`. Some parsers depend on tesseract which also needs to be installed manually.

Otherwise it LGTM :-)

File: parsers.dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Container image that executes the parser tests 
+# Build with "$ docker build -f parsers.dockerfile -t parsers . "
+# Run with  "$ docker run parsers US-CAL-CISO production"
+# If needed, pass any token env var using "docker run -e TOKEN=<token>"
+
+FROM python:3.8
+WORKDIR /workspace
+RUN apt-get update && apt-get install -y python3-opencv
```
Comment (FelixDQ) [inline parsers.dockerfile:8]: Ah, I missed that one. Thanks, @VIKTORVAV99 

```suggestion
RUN apt-get update && apt-get install -y python3-opencv tesseract-ocr tesseract-ocr-jpn tesseract-ocr-eng libgl1
```

**Comment by steren** [comment]
Sounds good, I added these additional modules

**Review by FelixDQ — APPROVED** [review]

**Pull request merged**